package.json

```diff
@@ -63,6 +72,7 @@
     "nsp": "^3.1.0",
     "nyc": "^11.3.0",
     "pre-commit": "^1.2.2",
+    "proxyquire": "^1.8.0",
```

Comment:
duplicate dependency functionality with mock-require
please remove proxyquire and refactor your tests to use mock-require